art/attacks/evasion/pixel_threshold.py

```diff
@@ -150,13 +151,21 @@ def generate(self, x: np.ndarray, y: Optional[np.ndarray] = None, **kwargs) -> n
 
         if self.th is None:
             logger.info(
-                "Performing minimal perturbation Attack. This takes substainally long time to process. For sanity check, pass th=10 to the Attack instance."
+                "Performing minimal perturbation Attack. \
+                This takes substainally long time to process. \
+                For sanity check, pass th=10 to the Attack instance."
             )
 
+        # NOTE: Pixel and Threshold Attacks are well defined for unprocessed images where the pixel values are,
+        #       8-Bit color i.e., the pixel values are np.uint8 in range [0, 255].
+
+        # TO-DO: Better checking of input image.
+        #        All other cases not tested needs the images to be rescaled to [0, 255].
         if np.max(x) <= 1:
```

Comment:
Can this checking be better? 

I thought to use dtype of x to check but the input can also be in float as in [0.0, 255.0]. So not sure about it.

Reply:
I would make it more explicit like:
```suggestion
        if self.estimator.clip_values[1] != 255.0:
```

Reply:
I think we should add check for `clip_values` in `_check_params`